teps/0061-allow-custom-task-to-be-embedded-in-pipeline.md

```diff
@@ -121,7 +121,23 @@ to adding support for `Run.RunSpec.Spec` the validations will be changed to supp
 "One of `Run.RunSpec.Spec` or `Run.RunSpec.Ref`" only and not both as part of a single
 API request to kubernetes.
 
-Structure of `RunSpec` after adding the field `Spec` of type `*v1beta1.EmbeddedTask`,
+Introducing a new type `v1alpha1.EmbeddedRunSpec`
```

Comment:
It would be great to also record the original design for history's sake. Maybe keep it as an Alternative? Something like:

```
## Alternatives

Use `v1beta1.EmbeddedTask` as `RunSpec.Spec` so that we
don't have to introduce a new embedded Spec type for runs.

Cons:
* brings some `PipelineTask`-specific fields (like PipelineResources)
  that don't have a use case in Runs yet.

Reply:
done !